app/controllers/jellyfish_demo/application_controller.rb

```diff
@@ -1,4 +1,12 @@
 module JellyfishDemo
   class ApplicationController < ::ApplicationController
+    rescue_from Fog::Compute::Demo::Error, with: :demo_error
```

Comment:
This appears to have been copied from another location. Fog isn't being used by the demo extension.